### DISCUSSION TITLE
Deleted a pro account which deleted a redis database that was on production is there a way to restore it?

### DISCUSSION THREAD

**Discussion by ozgurozkan123** [vercel/community#6353] [Help]
### Summary

Hello is there a way to restore a redis database it's deleted after a Pro team is deleted.

### Steps to Reproduce

Is there a way to restore the database?

**Comment by amyegan** [comment]
Hey @ozgurozkan123. Vercel KV does not currently support Upstash backup/restore features. Currently the only way to do this is with the [`redis-cli`](https://redis.io/docs/connect/cli/) package.

https://vercel.com/docs/storage/vercel-kv/limits

**Reply by tomatac** [reply]
How do we actually use redis-cli for backup?
SAVE and BGSAVE return errors:
(error) ERR Command is not available: 'SAVE'. See https://vercel.com/docs/storage/vercel-kv/vercel-kv-error-codes for details
